lib/fxpay/products.js

```diff
@@ -92,13 +174,53 @@
   //
 
 
-  function expandInfo(ob) {
-    return {
+  function buildProductReceiptMap() {
```

Comment:
Wondering if we could use something like this https://www.npmjs.com/package/jsdoc-rst-template + sphinx + rtd to add API docs to the code. I could see that being useful. It would also be handy for reviews too.